web/app/view/BaseMap.js

```diff
@@ -82,26 +86,39 @@ Ext.define('Traccar.view.BaseMap', {
             view: this.mapView
         });
 
-        target = this.map.getTarget();
-        if (typeof target === 'string') {
-            target = Ext.get(target).dom;
-        }
+        target = this.map.getTargetElement();
+        this.map.on('pointermove', function (evt) {
+            target.style.cursor = this.hasFeatureAtPixel(evt.pixel) ? 'pointer' : '';
+        });
 
-        this.map.on('pointermove', function (e) {
-            var hit = this.forEachFeatureAtPixel(e.pixel, function (feature, layer) {
-                return true;
-            });
-            if (hit) {
-                target.style.cursor = 'pointer';
-            } else {
-                target.style.cursor = '';
-            }
+        Ext.create('Ext.Component', {
```

Comment:
I don't understand, can you give an example?
do you mean something like this:

var popupDiv = document.createElement("div"); 
popupDiv.id = "popup";
document.body.appendChild(popupDiv);